README.md

```diff
@@ -56,6 +49,8 @@ Mobile First is a project that helps test and learn and the basic set up of an h
 ### 🚀 Live Demo <a name="live-demo"></a>
 You can access the live demo here: <a href="https://ddiiro.github.io/mobileFirst/">Ddiiro's Personal Portfolio</a>
```

Comment:
- [ ] you did a great job 😃 at hosting the website, but I see that when I click on the link to see the demo of the project the link of the demo redirects me to the page not found, so please 😊try to put the correct one to let the user see your portfolio or if you didn't learn the Github pages you can remove this link now until the lesson of deploy the website using the GitHub pages come and add it when you learn this lesson correctly and deploy your project 🚀